src/components/swap/SwapHeader.tsx

```diff
@@ -12,9 +12,15 @@ const StyledSwapHeader = styled.div`
   color: ${({ theme }) => theme.text2};
 `
 
-export default function SwapHeader({ allowedSlippage }: { allowedSlippage: Percent }) {
+export default function SwapHeader({
+  allowedSlippage,
+  className = '',
```

Comment:
Curious why we use empty string as the default when `className` is optional and can be `undefined`

Reply:
Fixed